langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -53,6 +53,13 @@ public type Text xml;
 
 # Returns number of xml items in an xml value.
 #
+# ```ballerina
+# xml `<book><title>Ballerina</title></book><book><title>Java</title></book>`.length() ⇒ 2
+# 
+# xml `<name>John</name><age>20</age>
+#       <roles><role>admin</role><role>teacher</role></roles>`.length() ⇒ 4
```

Comment:
Why do we need two examples? Is this specifically to show the length with whitespace (`xml:Text`)? Even then, does it have to be this lengthy?

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/9dc46b7c36c28b92b9ce360ed71621b3a6eff8c0